sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -259,62 +258,63 @@ func cloudauthAccountFromResourceData(data *schema.ResourceData) *v2.CloudauthAc
 		component := &cloudauth.AccountComponent{}
 
 		for key, value := range resourceComponent {
-			switch key {
-			case "type":
-				component.Type = cloudauth.Component(cloudauth.Component_value[value.(string)])
-			case "instance":
-				component.Instance = value.(string)
-			case "cloud_connector_metadata":
-				cloudConnectorMetadata := &cloudauth.CloudConnectorMetadata{}
-				if err := protojson.Unmarshal([]byte(value.(string)), cloudConnectorMetadata); err == nil {
+
+			if value != nil {
```

Comment:
@raghavanandan - Just the `nil` condition check hadn't worked for me, `value` may not be `nil` but it could be empty "" by default. can we please update this to - `if value != nil && value.(string) != ""` ?